cms/djangoapps/contentstore/views/item.py

```diff
@@ -1021,17 +1024,23 @@ def _create_xblock_child_info(xblock, course_outline, graders, include_children_
                 child, include_child_info=True, course_outline=course_outline,
                 include_children_predicate=include_children_predicate,
                 parent_xblock=xblock,
-                graders=graders
+                graders=graders,
+                user=user
             ) for child in xblock.get_children()
         ]
     return child_info
 
 
-def _get_release_date(xblock):
+def _get_release_date(xblock, user=None):
     """
     Returns the release date for the xblock, or None if the release date has never been set.
     """
     # Treat DEFAULT_START_DATE as a magic number that means the release date has not been set
+    if xblock.start.year < 1900 and user:
+        value = datetime(year=2030, month=1, day=1, tzinfo=UTC)
+        setattr(xblock, 'start', value)
```

Comment:
I think the better thing is just to set the start date to `DEFAULT_START_DATE` if the start year is before 1900